lib/bmg/relation.rb

```diff
@@ -105,6 +105,19 @@ def to_json(*args, &bl)
       to_a.to_json(*args, &bl)
     end
 
+    # Writes the relation data to CSV.
+    #
+    # `output` and `options` are what CSV::new itself
+    # recognizes, default options are CSV's.
+    #
+    # When no output is used, the method uses a string.
+    #
+    # The method always returns the output.
+    def to_csv(options = {}, output = nil)
+      options, output = {}, options unless options.is_a?(Hash)
```

Comment:
Maybe use the same arg names as `CSV.new`? So `string_or_io` instead of `output`.